custom_components/plugwise/manifest.json

```diff
@@ -1,13 +1,13 @@
 {
   "domain": "plugwise",
-  "name": "Plugwise Smile/Stretch Beta",
-  "after_dependencies": ["zeroconf"],
+  "name": "Plugwise Beta",
   "codeowners": ["@CoMPaTech", "@bouwew"],
   "config_flow": true,
   "documentation": "https://github.com/plugwise/plugwise-beta",
   "integration_type": "hub",
   "iot_class": "local_polling",
   "loggers": ["plugwise"],
   "requirements": ["plugwise==0.38.3"],
-  "version": "0.51.6"
+  "version": "0.51.6",
+  "zeroconf": ["_plugwise._tcp.local."]
```

Comment:
My first response would be: don't add this line! Pw-beta counts on the Core Plugwise integration doing this.
But you're right, if the Core Plugwise integration has never been loaded before, this is probably needed.